lib/readwise/client.rb

```diff
@@ -96,16 +106,50 @@ def get_export_page(page_cursor: nil, updated_after: nil, book_ids: [])
       params['updatedAfter'] = updated_after if updated_after
       params['ids'] = book_ids if book_ids.any?
       params['pageCursor'] = page_cursor if page_cursor
-      url = 'https://readwise.io/api/v2/export/?' + URI.encode_www_form(params)
-      export_uri = URI.parse(url)
-      export_req = Net::HTTP::Get.new(export_uri)
-      export_req['Authorization'] = "Token #{@token}"
-      export_res = Net::HTTP.start(export_uri.hostname, export_uri.port, use_ssl: true) do |http|
-        http.request(export_req)
-      end
-      raise Error, 'Export request failed' unless export_res.is_a?(Net::HTTPSuccess)
+      url = BASE_URL + 'export/?' + URI.encode_www_form(params)
 
-      JSON.parse(export_res.body)
+      get_readwise_request(url)
+
+    end
+
+    def transform_highlight(res)
```

Comment:
I think this method should be used in line 70 of `export_page`

Reply:
Great catch thank you. I figured I'd adopt the same pattern for the Tag data as well, lmk what you think. Apologies for this taking a bit, work's been pretty busy lol.

Reply:
There's no rush at all! This is free-time stuff.

Reply:
> figured I'd adopt the same pattern for the Tag data as well

Prefect!